components/game/game.js

```diff
@@ -40,10 +41,20 @@ const Game = () => {
     }
 
     /**
-     * @description End logic if hand has won 
+     * @description End logic if 3 hands have won, game over.
      * 
      */
     const checkWinner = hands.filter(player => player.hand.length === 0);
+
+    if (checkWinner.length === 3) {
+      setGameOverClause(true);
+      return;
+    }
```

Comment:
this is gonna need some tweaking. it got down to the point where I was out of cards, but nobody else was, and i had won the round but didn't have any cards to play and start the next round. we need to add some logic that would make it so that if the current round winner is out of cards, it moves onto the next player to start the round.